core/services/ocr2/plugins/ccip/internal/ccipdata/v1_0_0/commit_store.go

```diff
@@ -338,7 +338,8 @@ func (c *CommitStore) GetAcceptedCommitReportsGteTimestamp(ctx context.Context,
 	}
 
 	reportsQuery, err := query.Where(
-		c.address.String(),
+		// no key needed as logpoller.NewAddressFilter is used
+		"",
```

Comment:
wdyt about adding a Where function to log poller package which doesn't accept a key

Reply:
Past comments addressed here: [`7550e2b` (#14109)](https://github.com/smartcontractkit/chainlink/pull/14109/commits/7550e2b7721736099d277be946922cd73a962133)

Adding a Where function inside `logpoller` package sounds like a good idea. Let me do it

Reply:
Added here: [`1b19993` (#14109)](https://github.com/smartcontractkit/chainlink/pull/14109/commits/1b1999300bb408c114e88e5cbeb93fb031d5993b) and added this missing reference here: [`26420ac` (#14109)](https://github.com/smartcontractkit/chainlink/pull/14109/commits/26420ac5e37b103339d70904eac13a843813d2a1)